### PR TITLE
Allow overriding BootNodes

### PR DESCRIPTION
**Pull request by cammellos**
We need to allow the user to specify custom BootNodes, to address https://github.com/status-im/status-react/issues/4264 . 

The code has been changed so that it will use the provided ones if passed through config (`CusterConfig.BootNodes`) and is not empty. 

Otherwise fallback on default ones.

Important changes:

I have made it so that it will use the default ones if the user passes an empty array, not sure if that is something we want to support.

### CODE REVIEW

**Review by adriacidre — APPROVED** [review]
LGTM

**Review by dshulyak — DISMISSED** [review]
i dont think that this change is necessary

you can use `c.ClusterConfig.Enabled` on L563 to avoid overwrite

it will also setting static nodes but thats ok

UPD: i was wrong, it can't be false

File: geth/params/config.go [hunk 1]
```diff
@@ -582,7 +582,9 @@ func (c *NodeConfig) updateClusterConfig() error {
 
 	for _, cluster := range clusters {
 		if cluster.NetworkID == int(c.NetworkID) {
-			c.ClusterConfig.BootNodes = cluster.BootNodes
+			if len(c.ClusterConfig.BootNodes) == 0 {
```
Comment (dshulyak) [inline geth/params/config.go:585]: i am not sure what we really want here

if the goal is testing in isolated network then keeping static nodes doesn't make sense

if the goal is to avoid blockage - then removing our own bootnodes doesn't make sense, right? because how user can be sure for 100% if they are down/laggy or blocked

and i don't see a case when we want to overwrite one but keep another (talking about having static nodes)

Comment (cammellos) [reply geth/params/config.go:585]: The goal is to avoid blockage.
The use case is:

Say that status bootnodes IP addresses are blocked in Russia (seems like it might be already the case. We can be quite confident that they are, through user reporting, manual testing etc). 

The idea is to set up a new bootnode and send the IP addresses to the users, through a different channel ( twitter for example). The user will take care of updating the settings in the `status-app`.

Would that allow at least for a while the user to use status?

Basically we allow the user to choose bootnodes in advanced network settings.

I have to defer about static-nodes / bootnodes, as I don't know the difference, yet :)

Comment (dshulyak) [reply geth/params/config.go:585]: static nodes are just normal devp2p (whisper/les) peers, but app tries to connect with them all the time.
boot node serves as entry point to a distributed table of devp2p peers

my main confusion is that we overwrite boot nodes but leave static nodes, but tbh any other variant seems pretty the same. so i nothing against having it as is

Comment (cammellos) [reply geth/params/config.go:585]: @dshulyak thanks for the explanation, much clearer.

**Review by pedropombeiro — APPROVED** [review]
